app/views/simplified-account/settings/email-notifications/index.njk

```diff
@@ -1,14 +1,10 @@
-{% extends "../../../layout.njk" %}
+{% extends "../settings-layout.njk" %}
 
-{% block pageTitle %}
-  Settings - Email notifications
+{% block settingsPageTitle %}
+  Email notifications
 {% endblock %}
 
-{% block sideNavigation %}
-  {% include "../_settings-navigation.njk" %}
-{% endblock %}
-
-{% block mainContent %}
+{% block settingsContent %}
 
   <div class="govuk-grid-column-three-quarters">
```

Comment:
looks like this shouldn't be here, in your screenshots the layout is wrong. The settings-layout template already has the appropriate columns

Reply:
I can't where the layout in my screenshot is wrong? And are you sure, because the service name index.njk also has this [settingsContent](https://github.com/alphagov/pay-selfservice/blob/master/app/views/simplified-account/settings/service-name/index.njk#L11) line which is where I got it from.